### ISSUE TITLE
No tile yields in city view

### ISSUE THREAD

**Issue by CarpentierMartin** [Azurency/CQUI_Community-Edition#402]
**Describe the bug**
When in the city view, the tile yields are not showing anymore.

**To Reproduce**
1. Found a city
2. Click on city

**Expected behavior**
City's tile yields should be showing. (Even with the map option turned off)

R&F
No other mods

**Screenshots**
![image](https://user-images.githubusercontent.com/1552039/57584807-d31e0600-74df-11e9-8109-075989fba7ac.png)

**Comment by Sorxus** [comment]
I agree with you, even "ShowYieldsOnCityHover" option in file cqui_settings.sql doesn't do it's job either, there isn't any yield when set to "1". You have to turn it on manually with "Y" key (and off again)...

**Comment by Azurency** [comment]
@Sorxus Fixed yesterday in 1f0239b65611d689850cb732e9e420e635fa0e17

**Comment by Azurency** [comment]
Should be fixed now

**Comment by CarpentierMartin** [comment]
Sorry to reply so late but yes, it's been fixed since the last update on steam
